integration_tests/commands/async/json_test.go

```diff
@@ -1378,3 +1378,62 @@ func TestJsonARRTRIM(t *testing.T) {
 		})
 	}
 }
+
+func TestJsonSTRAPPEND(t *testing.T) {
+	conn := getLocalConnection()
+	defer conn.Close()
+
```

Comment:
What will the cleanup do here will it delete the doc key after each test?
Like this?
defer FireCommand(conn, "DEL doc")

Reply:
Yes, it can be using this function, or it can be using FLUSHDB command. This will ensure the next test which runs will have no conflict even if it uses the doc key